spec/homepage_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'homepage'
+
+describe Homepage do
+
+  it 'creates a new instance of Homepage class' do
+    expect(subject).to be_an_instance_of(Homepage)
+  end
+
+  describe '# display_restaurants' do
+    it 'prints a list of restaurants' do
+      expect(subject.display_restaurants).to be_an_instance_of(String)
+    end
+
+    # it 'should list restaurants with corresponding csv files' do
```

Comment:
I think that best practice is to delete redundant code and comments before pull requesting. (nit-picking)